src/sinks/splunk_hec/logs/integration_tests.rs

```diff
@@ -427,8 +427,12 @@ async fn splunk_auto_extracted_timestamp() {
         let (sink, _) = config.build(cx).await.unwrap();
 
         // With auto_extract_timestamp switched the timestamp comes from the message.
-        let message = "this message is on 2017-10-01 03:00:00";
-        let mut event = LogEvent::from(message);
+        // Note that as per <https://docs.splunk.com/Documentation/Splunk/latest/Data/Configuretimestamprecognition>
+        // the max age of timestamps is 2,000 days old. So we will test with a timestamp that
+        // is within that limit.
```

Comment:
🤔 if we aren't specifically trying to test logic around the 2k day limit, is there any reason we couldn't just use today ?